acceptance/detector_test.go

```diff
@@ -121,45 +121,6 @@ func testDetector(t *testing.T, when spec.G, it spec.S) {
 		})
 	})
 
-	when("assets env var is set on the builder", func() {
-		var copyDir, containerName string
-		it.Before(func() {
-			containerName = "test-container-" + h.RandString(10)
-			var err error
-			copyDir, err = ioutil.TempDir("", "test-docker-copy-")
-			h.AssertNil(t, err)
-		})
-		it.After(func() {
-			if h.DockerContainerExists(t, containerName) {
-				h.Run(t, exec.Command("docker", "rm", containerName))
-			}
-			os.RemoveAll(copyDir)
-		})
-
-		it("does not appear in the detection environment", func() {
```

Comment:
We could just flip the assertion instead of deleting the entire test... but, maybe the unit tests are sufficient. If we delete this, could we also back out this change? https://github.com/buildpacks/lifecycle/pull/591/files#diff-4d22a0bdf2091cd016a29e3d9de32343eb1460b757600af402decbf49a133d9e

Reply:
I'm fine keeping it, if desired.

My rationale for removing was that I didn't see any other tests about environment variables being present.

Reply:
Removed in latest commit.